README.md

```diff
@@ -49,6 +48,8 @@ Apply the plugin to each individual `com.android.application` module where you w
 apply plugin: 'com.xmartlabs.snapshot-publisher'
 ```
 
+> Note that Fabric's Maven Repository must to be added to the project because Android Snapshot Publisher plugin uses Fabric's plugin and it's hosted in their Maven Repository.
```

Comment:
```suggestion
> Note that Fabric's Maven Repository must be added to the project because Android Snapshot Publisher plugin uses Fabric's plugin and it's hosted in their Maven Repository.
```